pkg/koordlet/metriccache/api.go

```diff
@@ -26,6 +26,15 @@ type CPUMetric struct {
 	CPUUsed resource.Quantity
 }
 
+type GPUMetric struct {
+	CardIdx     int32             // index starting from 0
+	ID          string            // device UUID
+	Type        string            // product brand type of GPU
+	SMUtil      resource.Quantity // current utilization rate for the device
+	MemoryTotal resource.Quantity // total memory on the device, in bytes
```

Comment:
It's GPUMetric but contains MemoryTotal.. so why?

Reply:
to easily calculate memory utilization in the future feature😉

Reply:
you got me, removed